[2025-02-22 20:09:13,266][aiogram.dispatcher][INFO] - Start polling
[2025-02-22 20:09:13,625][aiogram.dispatcher][INFO] - Run polling for bot @YanixTradebot id=7420915283 - 'YanixTradeBot'
[2025-02-22 20:09:16,021][aiogram.event][INFO] - Update id=667609184 is not handled. Duration 0 ms by bot id=7420915283
[2025-02-22 20:09:16,021][aiogram.event][ERROR] - Cause exception while process update id=667609184 by bot id=7420915283
NameError: name 'result' is not defined
Traceback (most recent call last):
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 56, in __call__
    return await handler(event, data)
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 174, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 166, in _propagate_event
    response = await observer.trigger(event, **kwargs)
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/.venv/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/Users/richardgurtsiev/Desktop/projects/save/delete_2024/ml_service/yanix-trade/ynx_bot/routers/welcome_router.py", line 52, in cmd_start
    f'{result}\n\n'
NameError: name 'result' is not defined
[2025-02-22 20:09:25,021][aiogram.dispatcher][WARNING] - Received SIGINT signal
[2025-02-22 20:09:25,023][aiogram.dispatcher][INFO] - Polling stopped for bot @YanixTradebot id=7420915283 - 'YanixTradeBot'
[2025-02-22 20:09:25,023][aiogram.dispatcher][INFO] - Polling stopped
